include/openthread/config.h

```diff
@@ -35,6 +35,15 @@
 #ifndef OPENTHREAD_CONFIG_H_
 #define OPENTHREAD_CONFIG_H_
 
+#ifndef OPENTHREAD_POSIX
```

Comment:
Do we use this anywhere outside of CLI? Should we constrain this to a CLI-specific header?